
syntax = "proto3";


service Data_Send {
  rpc send_data (Data_msg_req) returns (Data_msg_res);
}


message Data_msg_req {
	string string_value = 1;
	int32 int_value = 2;

}

message Data_msg_res{
 	int32 status = 1;
	string status_name = 2;

}
